packages/flutter/lib/src/cupertino/form_section.dart

```diff
@@ -11,12 +11,6 @@ import 'list_section.dart';
 // iOS 14.2 SDK.
 const EdgeInsetsDirectional _kFormDefaultInsetGroupedRowsMargin = EdgeInsetsDirectional.fromSTEB(20.0, 0.0, 20.0, 10.0);
 
-// Standard header margin, determined from SwiftUI's Forms in iOS 14.2 SDK.
-const EdgeInsetsDirectional _kFormDefaultHeaderMargin = EdgeInsetsDirectional.fromSTEB(20.0, 16.0, 20.0, 10.0);
-
-// Standard footer margin, determined from SwiftUI's Forms in iOS 14.2 SDK.
-const EdgeInsetsDirectional _kFormDefaultFooterMargin = EdgeInsetsDirectional.fromSTEB(20.0, 0.0, 20.0, 10.0);
-
```

Comment:
I think the padding should be left in this file as is. It would be better to remove the padding in CupertinoListSection in `list_section.dart` and leave this one.

Reply:
I'm not sure if this solution will work properly. Because we can call `CupertinoListSection` without `CupertinoFormSection` so in this case we will not have paddings for header and footer at all

Reply:
I see what you are saying. After digging into this, it looks like #121543 may not actually be related to this, and may not actually be an issue.